bayes_kit/ess.py

```diff
@@ -39,7 +39,7 @@ def autocorr_np(chain: VectorType) -> VectorType:
     chain_ctr = chain - np.mean(chain)
     N = len(chain_ctr)
     acorrN = np.correlate(chain_ctr, chain_ctr, "full")[N - 1 :]
-    return acorrN / N
+    return np.asarray(acorrN / N)
```

Comment:
I think this can use the same style as the above change (line 25)

Reply:
done.